src/components/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { useContext, useMemo, useState } from "react";
+import "./ThemeSwitcher.scss";
+import classNames from "classnames";
+import { ThemeContext } from "./context/SwitchThemeContext";
+
+export const ThemeSwitcher = () => {
+  const { theme, toggleTheme } = useContext(ThemeContext);
+
+  useMemo(() => {
+    theme === "dark"
+      ? (document.body.style.backgroundColor = "#121721")
+      : (document.body.style.backgroundColor = "#f4f6f8");
+  }, [theme]);
```

Comment:
```suggestion
document.documentElement.style.setProperty("color-scheme", "dark");
```

Это пример, но все равно тебе это надо вынести в твой контекст